File: javascript/moderation_video_token_demo.js
/**
 * 视频审核服务token方式请求的使用示例
 */
var video = require("./moderation_sdk/moderation_video");
var token = require("./moderation_sdk/gettoken");
var utils = require("./moderation_sdk/utils");

// 初始化服务的区域信息，目前支持华北-北京(cn-north-4)、亚太-香港(ap-southeast-1)等区域信息
utils.initRegion("cn-north-4");

var username = "*******";        // 配置用户名
var domain_name = "*******";     // 配置用户名
var password = "*******";        // 密码

// obs链接需要和region区域一致，不同的region的obs资源不共享
demo_data_url = "https://obs-image-bj4.obs.cn-north-4.myhuaweicloud.com/bgm_recognition";

token.getToken(username, domain_name, password, function (token) {

    video.video(token, demo_data_url, 1, ["terrorism", "porn", "politics"], function (result) {
        console.log(result);
    })

});

File: python/moderation_video_token_demo.py
# -*- coding:utf-8 -*-
from moderation_sdk.gettoken import get_token
from moderation_sdk.moderation_video import moderation_video
from moderation_sdk.utils import init_global_env

if __name__ == '__main__':
    # Services currently support North China-Beijing(cn-north-4), Asia Pacific-Hong Kong(ap-southeast-1)
    init_global_env('cn-north-4')

    #
    # access asr, long_sentence，post data by token
    #
    user_name = '******'
    password = '******'
    account_name = '******'  # the same as user_name in commonly use

    demo_data_url = 'https://obs-image-bj4.obs.cn-north-4.myhuaweicloud.com/video_moderation.mp4'
    token = get_token(user_name, password, account_name)

    # call interface use the url
    result = moderation_video(token, demo_data_url, 1, ['porn', 'politics', 'terrorism'])
    print(result)